requirements-client.txt

```diff
@@ -1,5 +1,5 @@
-aiida-core==2.5.1
+aiida-core==2.5.2
 ase==3.23.0
-jarvis-tools==2024.4.10
+jarvis-tools==2024.4.30
```

Comment:
```suggestion
jarvis-tools==2024.4.10
```
Issue with `sklearn` not being installed.

Reply:
see https://github.com/usnistgov/jarvis/issues/322

Reply:
is this the way we should handle it, or via dependabot ignore?